galaxy.yml

```
@@ -1,14 +1,14 @@
 namespace: community
 name: aws
-version: 1.0.0
+version: 1.1.0
 readme: README.md
 authors:
   - Ansible (https://github.com/ansible)
 description: null
 license_file: COPYING
 tags: [community, aws, cloud, amazon]
 dependencies:
-  amazon.aws: '>=0.1.0'
+  amazon.aws: '>=1.0.0'
```

```suggestion
  amazon.aws: '>=1.1.0'
```

As far as I know we're testing amazon.aws:latest with community.aws:latest so it's probably safer to depend on 1.1.0 rather than 1.0.0